mountpoint-s3/src/prefetch.rs

```diff
@@ -92,6 +96,17 @@ where
     Prefetcher::new(part_stream, prefetcher_config)
 }
 
+pub type ParquetPrefetcher<Runtime> = Prefetcher<ClientPartStream<Runtime>>;
+
+// Creates an instance of the parquet-specific [Prefetch].
+pub fn parquet_prefetch<Runtime>(runtime: Runtime, prefetcher_config: PrefetcherConfig) -> ParquetPrefetcher<Runtime>
+where
+    Runtime: Spawn + Send + Sync + 'static,
+{
+    let part_stream = ClientPartStream::new(runtime);
+    Prefetcher::new(part_stream, prefetcher_config)
+}
```

Comment:
nit, make this a rustdoc so it shows up in documentation and IDEs

```suggestion
/// Creates an instance of the parquet-specific [Prefetch].
pub fn parquet_prefetch<Runtime>(runtime: Runtime, prefetcher_config: PrefetcherConfig) -> ParquetPrefetcher<Runtime>
where
    Runtime: Spawn + Send + Sync + 'static,
{
    let part_stream = ClientPartStream::new(runtime);
    Prefetcher::new(part_stream, prefetcher_config)
}
```